bundles/net.tourbook/src/net/tourbook/map2/view/TourMapPainter.java

```diff
@@ -1351,7 +1351,7 @@ private boolean doPaint_Pauses(final GC gcTile,
 
       if (tourData.isMultipleTours()) {
 
-         //TODO FB support joining pauses when creating a multiple tour
+         //TODO FB support joining pauses when creating a multiple tour, what is a multiple tour ?
```

Comment:
@wolfgang-ch I added the support to merge the pauses when concatenating tours but I don't know what a "multiple" tour is ? should the pauses be merged for multiple tours ?

Reply:
You have not yet used this feature ?

Multiple tours can be selected e.g. in the tourbook view.

![mt-multiple-tours](https://user-images.githubusercontent.com/1283445/93207193-e3d92e80-f75a-11ea-8ed2-ccf2ba192235.png)

Reply:
Oh you're saying that a multiple tour is when you select several tours in the tourbook view ? Then yes, I have used this feature but didn't know it had a name :-)
I thought a multiple tour was a tour that was merged.

Reply:
Fixed